x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -57,85 +57,79 @@ export function OverviewPage({ routeParams }: Props) {
   });
 
   return (
-    <>
-      {!hasAnyData && <LoadingObservability />}
```

Comment:
@cauemarcondes i think we need this loading state? WDYT? i guess there area already individual loading states on each component as well. That could be enabled.

Reply:
We still have it, I just moved it up to clean-up the main component. https://github.com/elastic/kibana/pull/79388/files/53ef5fe9687f62891f023c0cfffa875ee722c92f..7c6aa10e4fe925dfcd72d767b863f9da30ed78db#diff-e0ba82ac5e93f24f1eb755963db6aa93R46

Reply:
> i guess there area already individual loading states on each component as well. That could be enabled.

Yes, you are right, but the individual loading will only show up after `hasData` has been validated for each component. In the new implementation the `Loading Page` will last when hasData is no longer empty.

Reply:
got it, makes sense